### PR TITLE
Pioneer 20b glider ingestions

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP15MOAS-GL387/CP15MOAS-GL387_R00002_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00002/merged/cp_*.mrg,CP15MOAS-GL387-02-FLORTM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00003 recovery
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00002/merged/cp_*.mrg,CP15MOAS-GL387-03-CTDGVM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00003 recovery
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00002/merged/cp_*.mrg,CP15MOAS-GL387-04-DOSTAM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00003 recovery
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00002/merged/cp_*.mrg,CP15MOAS-GL387-05-PARADM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00003 recovery
```
Comment (rtravis-whoi) [inline CP15MOAS-GL387/CP15MOAS-GL387_R00002_ingest.csv:7]: looks good - readme file in raw data repo explains no data yet

File: CP15MOAS-PG564/CP15MOAS-PG564_D00001_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-PG564/D00001/merged-from-glider/cp_*.mrg,CP15MOAS-PG564-06-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP15MOAS-PG564/CP15MOAS-PG564_D00001_ingest.csv:8]: looks good

File: CP15MOAS-G0913/CP15MOAS-G0913_D00002_ingest.csv
Comment (reedan88) [inline CP15MOAS-G0913/CP15MOAS-G0913_D00002_ingest.csv:1]: Looks good.

File: CP15MOAS-GL379/CP15MOAS-GL379_D00003_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL379/CP15MOAS-GL379_D00003_ingest.csv:1]: Looks good.

File: CP15MOAS-GL379/CP15MOAS-GL379_R00002_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL379/CP15MOAS-GL379_R00002_ingest.csv:1]: Looks good

File: CP15MOAS-GL387/CP15MOAS-GL387_D00003_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL387/CP15MOAS-GL387_D00003_ingest.csv:1]: Looks good.

File: CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv:1]: Looks good

File: CP15MOAS-GL387/CP15MOAS-GL387_R00002_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL387/CP15MOAS-GL387_R00002_ingest.csv:1]: Looks good.

File: CP15MOAS-PG564/CP15MOAS-PG564_D00001_ingest.csv
Comment (reedan88) [inline CP15MOAS-PG564/CP15MOAS-PG564_D00001_ingest.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**